Jenkinsfile

```diff
@@ -87,6 +92,26 @@
     }
 }
 
+def startLogCatCollector() {
+    sh '''adb logcat -c
+    adb logcat > "logcat.txt" &
+    '''
+}
+
+def stopLogCatCollector(boolean archiveLog) {
```

Comment:
I think we should use a PID. this version is fragile as other poeple might be logged into the host and jobs might return unexpected results